### ISSUE TITLE
AlertDialog doesn't occupy the entire window space

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-jb#1483]
Compose 1.0.0-rc5
```
import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material.AlertDialog
import androidx.compose.material.Button
import androidx.compose.material.ExperimentalMaterialApi
import androidx.compose.material.Text
import androidx.compose.ui.Modifier
import androidx.compose.ui.window.Window
import androidx.compose.ui.window.application

@OptIn(ExperimentalMaterialApi::class)
fun main() = application {
    Window(onCloseRequest = ::exitApplication) {
        Column {
            Button({}) {}
            Button({}) {}
            Button({}) {}
            Button({}) {}
            Button({}) {}
            Button({}) {}
            Button({}) {}
            Button({}) {}
            Box(Modifier.fillMaxSize()) {
                AlertDialog(
                    onDismissRequest = {},
                    title = { Text("Title") },
                    text = { Text("Text") },
                    confirmButton = {
                        Button(
                            onClick = {}
                        ) {
                            Text("Confirm")
                        }
                    },
                    dismissButton = {
                        Button(
                            onClick = {}
                        ) {
                            Text("Dismiss")
                        }
                    }
                )
            }
        }
    }
}
```
Actual:
![image](https://user-images.githubusercontent.com/5963351/143769185-6d2c9b35-1391-493a-88d8-0fc62ef5c834.png)

Expected:
![image](https://user-images.githubusercontent.com/5963351/143769219-a61852f6-6273-429a-918f-08b5f4a76d30.png)

**Comment by igordmn** [comment]
False alarm, it was fixed after 1.0.0-beta5

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5147">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
